scripts/e2e-iOS.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+export SAUCE_USERNAME=okta-qa
+export SAUCE_ACCESS_KEY="$(aws s3 --quiet --region us-east-1 cp s3://ci-secret-stash/prod/saucelabs/saucelabs_access_key /dev/stdout)"
+export SAUCE_PLATFORM_NAME="iOS";
+export TRAVIS=true # work-around to run tests on saucelabs instead of chrome
+export TRAVIS_JOB_NUMBER=${TEST_SUITE_ID}
+export TRAVIS_BUILD_NUMBER=${TEST_SUITE_RESULT_ID}
+
+cd ${OKTA_HOME}/${REPO}
+
+# Download and start sauce connect
+curl -o ${OKTA_HOME}/${REPO}/sc-4.5.3-linux.tar.gz https://saucelabs.com/downloads/sc-4.5.3-linux.tar.gz
+tar -xzf ${OKTA_HOME}/${REPO}/sc-4.5.3-linux.tar.gz
+${OKTA_HOME}/${REPO}/sc-4.5.3-linux/bin/sc -u ${SAUCE_USERNAME} -k ${SAUCE_ACCESS_KEY} -i ${TRAVIS_JOB_NUMBER} &
+
+aws s3 --quiet --region us-east-1 cp s3://ci-secret-stash/prod/signinwidget/test_credentials ./test_credentials.yaml
+
+pip install yq
+
+export WIDGET_TEST_SERVER=$(cat ./test_credentials.yaml | yq .WIDGET_TEST_SERVER | tr -d '"')
+export WIDGET_BASIC_USER=$(cat ./test_credentials.yaml | yq .WIDGET_BASIC_USER | tr -d '"')
+export WIDGET_BASIC_PASSWORD=$(cat ./test_credentials.yaml | yq .WIDGET_BASIC_PASSWORD | tr -d '"')
+export WIDGET_BASIC_USER_2=$(cat ./test_credentials.yaml | yq .WIDGET_BASIC_USER_2 | tr -d '"')
+export WIDGET_BASIC_PASSWORD_2=$(cat ./test_credentials.yaml | yq .WIDGET_BASIC_PASSWORD_2 | tr -d '"')
+export WIDGET_BASIC_USER_3=$(cat ./test_credentials.yaml | yq .WIDGET_BASIC_USER_3 | tr -d '"')
+export WIDGET_BASIC_PASSWORD_3=$(cat ./test_credentials.yaml | yq .WIDGET_BASIC_PASSWORD_3 | tr -d '"')
+export WIDGET_BASIC_USER_4=$(cat ./test_credentials.yaml | yq .WIDGET_BASIC_USER_4 | tr -d '"')
+export WIDGET_BASIC_PASSWORD_4=$(cat ./test_credentials.yaml | yq .WIDGET_BASIC_PASSWORD_4 | tr -d '"')
+export WIDGET_BASIC_USER_5=$(cat ./test_credentials.yaml | yq .WIDGET_BASIC_USER_5 | tr -d '"')
+export WIDGET_BASIC_PASSWORD_5=$(cat ./test_credentials.yaml | yq .WIDGET_BASIC_PASSWORD_5 | tr -d '"')
```

Comment:
super-nit: Might be easier to maintain in the long term if you loop over a list of known environment variables, and run this command instead of running the same command on multiple lines.